### ISSUE TITLE
Just do "Get Gelocation id" one time, not on every Iteration-Cyrcle

### ISSUE THREAD

**Issue by baerengraben** [baerengraben/ioBroker.swiss-weather-api#41]
The new API limits the Calls to 50/day. So we need to be sparing with the requests. the geolocation id actually only needs to be read the first time for requesting the forecasts. After that, the value can be used again for each forecast request.

**Comment by cdellasanta** [comment]
I wanted to write a new issue, but this one I think this explains the behaviour i wanted to report: 
![image](https://user-images.githubusercontent.com/70055566/119254025-533dd280-bbb4-11eb-9863-6d1255fc2a33.png)

**Comment by baerengraben** [comment]
Hi @cdellasanta 
Per iteration cycle I did 3 requests:

- getToken
- get geolocationId (is needed for getting forecast)
- get forecast (delivers the final forecast)

So I do 3 calls per iteration. Where I assume that the "getToken" call is not really counted as a request. 

I did a fix today. now the geolocation id is read only 1x and reused afterwards. So in the actual version (current master branch in Github) 49 forecast calls should be possible. I am currently testing this and, if successful,  I will make a new release 0.9.1.
